keyboards/ploopyco/ploopyco.c

```diff
@@ -127,30 +130,26 @@ void encoder_driver_task(void) {
     lastScroll = timer_read();
 }
 #endif
-
 report_mouse_t pointing_device_task_kb(report_mouse_t mouse_report) {
     if (is_drag_scroll) {
-        scroll_accumulated_h += (float)mouse_report.x / PLOOPY_DRAGSCROLL_DIVISOR_H;
-        scroll_accumulated_v += (float)mouse_report.y / PLOOPY_DRAGSCROLL_DIVISOR_V;
-
-        // Assign integer parts of accumulated scroll values to the mouse report
-        mouse_report.h = (int8_t)scroll_accumulated_h;
+        // Assign integer parts of accumulated scroll values with remainders to the mouse report
+        mouse_report.h = (mouse_report.x + scroll_remainder_h) / PLOOPY_DRAGSCROLL_DIVISOR_H;
 #ifdef PLOOPY_DRAGSCROLL_INVERT
-        mouse_report.v = -(int8_t)scroll_accumulated_v;
+        mouse_report.v = -(mouse_report.y + scroll_remainder_v)  / PLOOPY_DRAGSCROLL_DIVISOR_V;
```

Comment:
This method avoids conversions to and from floats which can introduce inaccuracies when values are accumulated and integer math is faster.

Reply:
Less float math is better, especially for the atmega32u4 based boards!